README.md

```diff
@@ -4,6 +4,8 @@
 
 > It is now a requirement for clusters to run Kubernetes >=1.19.
 
+> override labels with unregistered `kubernetes.io` annotations will be deprecated. It'll soon be a requirement to use `kubeaudit.io` instead.
+Refer to this [discussion](https://github.com/Shopify/kubeaudit/issues/457) for additional context.
```

Comment:
when should we actually deprecate them?

Reply:
We could tie it to the k8s release and do it when 1.28 is released? That would give users about 6 months before the change

Reply:
sounds like a plan! great idea! :)